pkg/distro/BUILD

```diff
@@ -25,6 +26,13 @@ pkg_tar(
     strip_prefix = ".",
 )
 
+genrule(
+    name = "small_workspace",
+    srcs = ["//:WORKSPACE"],
+    outs = ["WORKSPACE"],
+    cmd = "sed -e '/### INTERNAL ONLY/,$$d' $(location //:WORKSPACE) >$@",
```

Comment:
My understanding from the [documentation](https://docs.bazel.build/versions/master/build-ref.html#workspace) is that it is not strictly necessary to strip these out unless commands are going to be run from within the `rules_pkg` workspace itself, which I imagine to be rare at best.  Does the CI testing do this or need a "complete" WORKSPACE somehow?

The release WORKSPACE could also be empty if it's otherwise not used, if `pkg/deps.bzl` was kept up-to-date.

OTOH, keeping this `sed` call in could also help with documentation.  What do you think?

Reply:
The intent is that if you expanded the distribution tarball and did `bazel test ...` it would work.
We don't need skydoc because the distro does not include the distro folder where we invoke skydoc.

That aside, I think you are getting to the seeming strangeness of this PR. This is not for the benefit of normal users. It is because I am trying to get the Bazel integration tests to work with this repo rather than the (obsolete) pkg_tar that they use. I can't explain why the external repos all require their own WORKSPACE - the testing infrastructure is too spaghetti to follow - but this seems to help.

Reply:
> The intent is that if you expanded the distribution tarball and did `bazel test ...` it would work.
> We don't need skydoc because the distro does not include the distro folder where we invoke skydoc.

Fair enough.  I was wondering about something like this, but at least as 0.2.6, tests are excluded from the release tarball.  Are there plans to add them in?  They're relatively small at this point, so I don't see why not.

> That aside, I think you are getting to the seeming strangeness of this PR. This is not for the benefit of normal users. It is because I am trying to get the Bazel integration tests to work with this repo rather than the (obsolete) pkg_tar that they use. I can't explain why the external repos all require their own WORKSPACE - the testing infrastructure is too spaghetti to follow - but this seems to help.

Ah, OK.  Yay code pasta, I guess.

With all of these combined, the intent is clear.  Thanks!